src/cascadia/TerminalApp/WindowsPackageManagerFactory.h

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// Module Name:
+// - WindowsPackageManagerFactory.h
+//
+// Abstract:
+// - This factory is designed to create production-level instances of WinGet objects.
+//   Elevated sessions require manual activation of WinGet objects and are not currently supported,
+//   while non-elevated sessions can use the standard WinRT activation system.
+// Author:
+// - Carlos Zamora (carlos-zamora) 23-Jul-2024
+
+#pragma once
+
+#include "../../types/inc/utils.hpp"
+#include <combaseapi.h>
+#include <wil/result_macros.h>
+#include <wil/win32_helpers.h>
+#include <type_traits>

```

Comment:
I suspect including at least some these is not necessary since they should be part of the PCH file.